h2o-core/build.gradle

```diff
@@ -11,6 +11,8 @@ dependencies {
   // Required for h2o-core
   compile "joda-time:joda-time:2.3"
   compile "gov.nist.math:jama:1.0.3"
+  compile 'com.github.fommil.netlib:all:1.1.2'
```

Comment:
Should these dependencies be in h2o-algos? Are they used in h2o-core?

Reply:
@michalkurka No, you are right. I placed them in `h2o-core/build.gradle` because Jama was there before. And I just followed. But I also don't see any reason to have it in `h2o-core`. So I have moved it to `h2o-algos/build.gradle`